spam/01_spam_tutorial.py

```diff
@@ -53,9 +67,10 @@
 # ### Data Splits in Snorkel
 #
 # We split our data into 4 sets:
-# * **Training Set**: The largest split of the dataset. We do not have ground truth or "gold" labels for these data points; we will be generating their labels with weak supervision.
-# * \[Optional\] **Development Set**: A small labeled subset of the training data (e.g. 100 points) to guide LF iteration. See note below.
-# * **Validation Set**: A labeled set used to tune hyperparameters and/or perform early stopping while training the classifier.
+# * **Training Set**: The largest split of the dataset, and the one without ground truth ("gold") labels.
```

Comment:
nit: do we need the "gold labels" terminology? we can be consistent and say ground ruth everywhere

Reply:
it's used quite a bit in the snorkel repo doc and papers/blogs. i think it's ok to leave here.